Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -782,6 +759,24 @@ extension SettingsTableViewController: DailyValueScheduleTableViewControllerDele
     }
 }
 
+extension SettingsTableViewController: GlucoseRangeScheduleStorageDelegate {
+    func saveSchedule(for viewController: GlucoseRangeScheduleTableViewController, completion: @escaping (SaveGlucoseRangeScheduleResult) -> Void) {
```

Comment:
[nit] One of these `saveSchedule` functions pulls values-to-be-saved directly from the `viewController` while the other passes those values via a function parameter. Might want to eventually make these consistent. (I'd go with the function parameter option. That way, no inherent knowledge of the view controller is necessary.)